modules/nextflow/src/main/groovy/nextflow/executor/BridgeExecutor.groovy

```diff
@@ -0,0 +1,179 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.executor
+import java.nio.file.Path
+import java.util.regex.Pattern
+
+import groovy.util.logging.Slf4j
+import nextflow.processor.TaskRun
+/**
+ * Processor for BRIDGE resource manager (DRAFT)
+ *
+ * See https://github.com/cea-hpc/bridge 
+ *
+ *
+ * @author Eric Bonnet <eric.d.bonnet@gmail.com>
+ */
+@Slf4j
+class BridgeExecutor extends AbstractGridExecutor {
+
+    //  submission pattern example: Submitted Batch Session 1277017
+    static private Pattern SUBMIT_REGEX = ~/Submitted Batch Session (\d+)/
+
+    /**
+     * Gets the directives to submit the specified task to the cluster for execution
+     *
+     * @param task A {@link TaskRun} to be submitted
+     * @param result The {@link List} instance to which add the job directives
+     * @return A {@link List} containing all directive tokens and values.
+     */
+    protected List<String> getDirectives(TaskRun task, List<String> result) {
+
+        String job_name = ""
+        // parenthesis are not compatible with bridge submission commands
+        job_name = getJobNameFor(task)
+        job_name = job_name.replace("(", "")
+        job_name = job_name.replace(")", "")
+
+        result << '-r' << job_name 
+        result << '-o' << quote(task.workDir.resolve(TaskRun.CMD_LOG)) 
+
+        // number of cores per parallel task to allocate 
+        if( task.config.cpus > 1 ) {
+            result << '-c' << task.config.cpus.toString()
+        }
+
+        // maximum walltime of the batch job in seconds
+        if( task.config.time ) {
+            result << '-T' << task.config.getTime().toSeconds() 
+        }
+
+        // maximum memory amount required per allocated core in Mo (default is chosen by the underlying system)
+        if( task.config.getMemory() ) {
+            result << '-M' << task.config.getMemory().toMega().toString() 
+        }
+
+        // the requested partition (a.k.a queue) name
+        if( task.config.queue ) {
+            result << '-q' << (task.config.queue.toString())
+        }
+
+        // other cluster options 
+        if( task.config.clusterOptions ) {
+            result << task.config.clusterOptions.toString() << ''
+        }
+
+        return result
+    }
+
+    String getHeaderToken() { '#MSUB' }
+
+    /**
+     * The command line to submit this job
+     *
+     * @param task The {@link TaskRun} instance to submit for execution to the cluster
+     * @param scriptFile The file containing the job launcher script
+     * @return A list representing the submit command line
+     */
+    @Override
+    List<String> getSubmitCommandLine(TaskRun task, Path scriptFile ) {
+
+        ['ccc_msub', scriptFile.getName()]
+
+    }
+
+    /**
+     * Parse the string returned by the {@code sbatch} command and extract the job ID string
```

Comment:
Should be `ccc_msub` instead of `sbatch` I assume.

Reply:
Done !